pkg/lib/indexer/indexer.go

```diff
@@ -513,38 +523,72 @@ func (i ImageIndexer) ExportFromIndex(request ExportFromIndexRequest) error {
 	}
 
 	var errs []error
+	var g errgroup.Group
```

Comment:
Can `errs` and `mu` be dropped altogether? The original error handling behavior is suspect (especially considering original L532, which I think is actually looking at the `err` that was assigned on original L504?), and I would be easier to understand if this method were to only return the first error it encounters (which is consistent with errgroup's behavior).

Reply:
same comment as [here](https://github.com/operator-framework/operator-registry/pull/429#discussion_r485815618)